conda/recipes/versions.yaml

```diff
@@ -15,7 +15,7 @@ xgboost_version:
 conda_version:
   - '=4.10.3'
 conda_build_version:
-  - '=3.20.3'
+  - '=3.21.7'
```

Comment:
Is there a reason the `conda_build_version` variable is being updated in this PR?

Reply:
conda_build caused a build error.  3.20.3 doesn't have a python 3.9 version.